tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3102,7 +3102,7 @@ def _ssim_helper(x, y, reducer, max_val, compensation=1.0, k1=0.01, k2=0.03):
     compensation: Compensation factor. See above.
     k1: Default value 0.01
     k2: Default value 0.03 (SSIM is less sensitivity to K2 for lower values, so
-      it would be better if we taken the values in range of 0< K2 <0.4).
+      it would be better if we took the values in the range of 0< K2 <0.4).
```

Comment:
`0 < K2 < 0.4` since you fixed spacing on `None,None,None` above

Reply:
made required changes